cupy/__init__.py

```diff
@@ -133,8 +133,8 @@ def is_available():
 from numpy import half  # NOQA
 from numpy import single  # NOQA
 from numpy import double  # NOQA
-from numpy import float_  # NOQA
-from numpy import longfloat  # NOQA
+from numpy import float64 as float_  # NOQA
+# from numpy import longfloat  # NOQA   # XXX
```

Comment:
What is the reason that this is commented out?

Reply:
IIUC, `longfloat` is a (np2 removed) alias for `longdouble`. The latter resolves to either float96 or float128, none of which was ever supported by CuPy, again IIUC.


```
In [1]: import numpy as np

In [2]: np.finfo(np.longfloat)
Out[2]: finfo(resolution=1e-18, min=-1.189731495357231765e+4932, max=1.189731495357231765e+4932, dtype=float128)

In [3]: np.longfloat
Out[3]: numpy.longdouble

In [4]: np.__version__
Out[4]: '1.26.4'
```

and


```
In [1]: import numpy as np

In [2]: np.__version__
Out[2]: '2.0.0rc2'

In [3]: np.longfloat
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
Cell In[3], line 1
----> 1 np.longfloat

File ~/mambaforge/envs/np2.0/lib/python3.11/site-packages/numpy/__init__.py:397, in __getattr__(attr)
    394     raise AttributeError(__former_attrs__[attr])
    396 if attr in __expired_attributes__:
--> 397     raise AttributeError(
    398         f"`np.{attr}` was removed in the NumPy 2.0 release. "
    399         f"{__expired_attributes__[attr]}"
    400     )
    402 if attr == "chararray":
    403     warnings.warn(
    404         "`np.chararray` is deprecated and will be removed from "
    405         "the main namespace in the future. Use an array with a string "
    406         "or bytes dtype instead.", DeprecationWarning, stacklevel=2)

AttributeError: `np.longfloat` was removed in the NumPy 2.0 release. Use `np.longdouble` instead.

```